webapp2.py

```diff
@@ -32,18 +31,18 @@
 import sys
 import threading
 import traceback
-import urllib
 from wsgiref import handlers
 
+import six
```

Comment:
This changes adds a new dependency on six. Can you accomplish this without adding six? (we can still use six in the test suite, I'd just prefer if this library didn't take on any additional run-time dependencies).